chore: update bug report template (#1675)
ipfs-shipyard/ipfs-desktop: .github/ISSUE_TEMPLATE/bug_report.md

ATTACH LOGS

If possible:

> 1. From the "IPFS icon" menu in your menubar/system tray, go to `Advanced` → `Open Logs Directory` *(for Linux users who do not have this menu, skip these steps or look for your logs directory at `...../.config/IPFS Desktop` )*

Reviewer: ```suggestion
1. From the "IPFS icon" menu in your menubar/system tray, go to `Advanced` → `Open Logs Directory` *(for Linux users who do not have this menu, logs can be found in `$HOME/.config/IPFS Desktop` )*
```
